llvm/CMakeLists.txt

```diff
@@ -534,6 +534,10 @@ else()
   option(LLVM_ENABLE_THREADS "Use threads if available." ON)
 endif()
 
+set(LLVM_ENABLE_ICU "ON" CACHE STRING "Use ICU for character conversion support if available. Can be ON, OFF, or FORCE_ON")
```

Comment:
The description says this can be FORCE_ON, but you didn't actually implement that.

Reply:
Thanks I have implemented this